ui/store/actions.ts

```diff
@@ -4515,6 +4516,20 @@ export function setFirstTimeUsedNetwork(chainId: string) {
   return submitRequestToBackground('setFirstTimeUsedNetwork', [chainId]);
 }
 
+export function setOpenSeaTransactionSecurityProviderPopoverHasBeenShown(): ThunkAction<
+  void,
+  MetaMaskReduxState,
+  unknown,
+  AnyAction
+> {
+  return async (dispatch) => {
+    await submitRequestToBackground(
+      'setOpenSeaTransactionSecurityProviderPopoverHasBeenShown',
+    );
+    await forceUpdateMetamaskState(dispatch);
```

Comment:
Can you plz detail why is this required `await forceUpdateMetamaskState(dispatch);` ?

Reply:
Good question and good catch! I removed it now and everything works fine. Not sure why I added it before, like something wasn't working properly and I was looking into other actions that were implemented in the same way...
It is fixed now: 85964c7c518d262946fc6d109f44c9d4863e23b0